CONTRIBUTING.md

```diff
@@ -0,0 +1,49 @@
+# Developer guidelines
+
+<!-- vim-markdown-toc GFM -->
+
+- [Development](#development)
+  - [Dev environment set up](#dev-environment-set-up)
+- [Pull requests](#pull-requests)
+  - [Stages: Draft and Ready for review](#stages-draft-and-ready-for-review)
+
+<!-- vim-markdown-toc -->
+
+## Development
+
+### Dev environment set up
+
+```bash
+# Create an .env based on the example
+$ cp .env.example .env
+
+# Start MySQL in docker
+$ docker compose up -d
+# Wait for the db to finish starting, it takes a few seconds before it's ready...
+
+# Setup the database
+$ npm run setup
+
+# Start the app, which will be running on localhost:3000
+$ npm run dev
+
+# Visually see the database in a GUI:
+$ npm run db:studio
+```
+
+Note `npm run setup` creates a local `mysql-data/` directory, and is a one time setup. That directory can be safely deleted if the database needs to be reset, or if errors occur when starting the database.
```

Comment:
You can use the new [Note Markdown syntax](https://github.com/community/community/discussions/16925) too if you want:

> **Note**
> `npm run setup` creates a local...

Reply:
Oh neat! let me give that a try